docker-compose.client.yml

```diff
@@ -0,0 +1,72 @@
+version: '3.5'
+# Use this script for running up nightfall_3 client in 'developer' mode with local
+# bindings.  See the readme for more information.
+services:
+  client1:
+    image: ghcr.io/eyblockchain/nightfall3-client:latest
```

Comment:
i think you should change the image name.... its strange that its called `eyblockchain`

Reply:
Yes, but this applies to all images that we are building now in the Dockerfiles in the root folder and not only this PR.
So we have to do it in all images I think.
- ghcr.io/eyblockchain/nightfall3-deployer:latest
- ghcr.io/eyblockchain/nightfall3-worker:latest
- ghcr.io/eyblockchain/nightfall3-client:latest
- ghcr.io/eyblockchain/nightfall3-optimist:latest
- ghcr.io/eyblockchain/nightfall3-proposer:latest
- ghcr.io/eyblockchain/nightfall3-user-local:latest